homeassistant/components/apple_tv/__init__.py

```diff
@@ -88,14 +88,18 @@ class AppleTVEntity(Entity):
     """Device that sends commands to an Apple TV."""
 
     _attr_should_poll = False
+    _attr_has_entity_name = True
+    _attr_name = None
 
     def __init__(self, name, identifier, manager):
         """Initialize device."""
         self.atv = None
         self.manager = manager
-        self._attr_name = name
         self._attr_unique_id = identifier
-        self._attr_device_info = DeviceInfo(identifiers={(DOMAIN, identifier)})
+        self._attr_device_info = DeviceInfo(
+            identifiers={(DOMAIN, identifier)},
+            name=name,
+        )
```

Comment:
Unrelated, but we likely have a lot of data we could add here

Reply:
Definitely, but I don't have access to those devices.

I'll note it down as an integration to check the tests/lib to see if I can fill this info using this info. But in a later PR ofc